activesupport/lib/active_support/testing/silence_stream.rb

```diff
@@ -0,0 +1,15 @@
+module ActiveSupport
+  module Testing
+    module SilenceStream
```

Comment:
+1, but I'd say we should go further, and add a comment noting how hopelessly unthreadsafe this is... it's not "it might change in the future" internal: it's "you will regret using this" internal.